pom.xml

```diff
@@ -467,6 +567,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.1.2</version>
+                <configuration>
```

Comment:
we should skip execution of plugin when tests are skiped, so if add goal `analyze-only`
please also add
```xml
<skip>${maven.test.skip}</skip>
```